app/controllers/RiderController.java

```diff
@@ -1,12 +1,20 @@
 package controllers;
 
+import com.fasterxml.jackson.databind.JsonNode;
+import models.Rider;
+import org.apache.commons.lang3.exception.ExceptionUtils;
 import play.core.j.HttpExecutionContext;
+import play.mvc.BodyParser;
 import play.mvc.Controller;
 import play.mvc.Result;
 import repository.interfaces.RiderRepository;
 
 import javax.inject.Inject;
+import java.util.Date;
+import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)